### PR TITLE
[Feature] 447882 - Active Messages Step 3

### PR DESCRIPTION
**Pull request by mpamaro**
This pull request aims to add new features and fixes to help the development of Active Messages in Blip Desk App.

- [DSDashedBorder] New widget that creates a dashed border container.
- [DSExpandedImage] New widget that show an imagem and opens up an image visualizer when tapped.
- [DSInputContainerShape] New enum that represents the shape of an input container.
- [DSVideoBody] New widget to video display (same layout as DSVideoMessageBubble, but without the bubble).
- [Typography] Added line height props to all styles 
- [Typography] Fixed text height distribution behavior.
- [Typography] Fixed linkify when using WidgetSpan.
- [DSBottomSheetCountries] Fixed icon background color.
- [DSSearchInput] Improved color props.
- [DSFileExtensionIcon] Added defaultColor prop.
- [DSTextField] Removed requirement of controller prop.
- [DSInputContainer] Added shape selector prop.
- [Sample] Fixed sample bottom sheet showcase name.
- [DSTextField] Changed from DSTextFormField to DSTextField.

### CODE REVIEW

File: lib/src/services/ds_file.service.dart [hunk 1]
```diff
@@ -1,14 +1,13 @@
-import 'dart:io' as io;
+import 'dart:io';
 
+import 'package:blip_ds/blip_ds.dart';
```
Comment (githubdoandre) [inline lib/src/services/ds_file.service.dart:3]: you can remove this import and use individual file imports

File: lib/src/widgets/borders/ds_dashed_border.widget.dart [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import 'package:dotted_border/dotted_border.dart';
+import 'package:flutter/material.dart';
+
+import '../../../blip_ds.dart';
```
Comment (githubdoandre) [inline lib/src/widgets/borders/ds_dashed_border.widget.dart:4]: you can remove this import and use individual file imports

File: lib/src/widgets/fields/ds_text_field.widget.dart [hunk 3]
```diff
@@ -0,0 +1,137 @@
+import 'package:blip_ds/src/enums/ds_input_container_shape.enum.dart';
```
Comment (githubdoandre) [inline lib/src/widgets/fields/ds_text_field.widget.dart:1]: you can use relative imports

https://dart.dev/tools/linter-rules#prefer_relative_imports

File: lib/src/widgets/borders/ds_dashed_border.widget.dart [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  });
+
+  final Color borderColor;
+  final double strokeWidth;
+  final Widget child;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.all(1),
```
Comment (PaollaLira) [inline lib/src/widgets/borders/ds_dashed_border.widget.dart:21]: By default, we write the double with the second digit after the dot, like this: 1.0

File: lib/src/widgets/borders/ds_dashed_border.widget.dart [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+      child: DottedBorder(
+        color: borderColor,
+        strokeWidth: strokeWidth,
+        radius: const Radius.circular(8),
```
Comment (PaollaLira) [inline lib/src/widgets/borders/ds_dashed_border.widget.dart:25]: By default, we write the double with the second digit after the dot, like this: 8.0

File: lib/src/widgets/borders/ds_dashed_border.widget.dart [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+        borderType: BorderType.RRect,
+        dashPattern: const [3, 3],
```
Comment (PaollaLira) [inline lib/src/widgets/borders/ds_dashed_border.widget.dart:27]: By default, we write the double with the second digit after the dot, like this: 3.0

**Review by PaollaLira — APPROVED** [review]

**Review by githubdoandre — APPROVED** [review]

**Pull request merged**